pitop/camera/camera.py

```diff
@@ -144,7 +146,7 @@ def stop_detecting_motion(self):
         self._frame_handler.remove_action(CaptureActions.DETECT_MOTION)
 
     @type_check
-    def start_handling_frames(self, callback_on_frame, frame_interval=1):
+    def start_handling_frames(self, callback_on_frame, frame_interval=1, opencv=False):
```

Comment:
i think that this is probably better:
```suggestion
    def start_handling_frames(self, callback_on_frame, frame_interval=1, format="PIL"):
```
and also support `format = "OpenCV"`